test/samples-js-extra/AddProductsNoPartialUpsert/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = async function (context) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    var products = [];
+    for(let i = 0; i < 1000; i++) {
+        products.add({"productId": i,
+        "name": "test"+i,
```

Comment:
In general if you're going to make functionality changes from the "original" ones they should be done explicitly. For example here the name is `test#` but in the C# ones it's only going to be `test` for every item. 

The implementations should be as exact as possible - so either update the "original" tests/samples/helpers with the new functionality or keep the implementation exactly the same across all supported languages.